### PR TITLE
Bug 1870622: Update topology list view to be more consistent with workloads page

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
**Fixes**
https://issues.redhat.com/browse/ODC-4418

**Description**
Update the topology list view to contain separate cells for memory and cpu metrics. Add polling for metric and alert updates. 

**Screenshots**

Before:
image

After:
image

**Browser conformance**: 
- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

/kind bug

cc @openshift/team-devconsole-ux @serenamarie125 @beaumorley @bgliwa01

### CODE REVIEW

**Comment by debsmita1** [comment]
@jeff-phillips-18 when the sidebar is open, the cells for memory & cpu usage don't show up. have a look at the below gif
listview

**Comment by jeff-phillips-18** [comment]
@debsmita1 Correct. That is by design as in the project workloads tab in the admin perspective.

**Comment by debsmita1** [comment]
verified this locally
/lgtm

File: frontend/packages/dev-console/src/components/topology/list-view/TopologyListView.tsx [hunk 1]
```diff
@@ -139,6 +172,91 @@ const TopologyListView: React.FC<TopologyListViewProps> = ({
...
+          if (_.includes([401, 403, 502, 503], status) && _.isEmpty(metrics)) {
+            throw new Error(`Could not fetch metrics, status: ${status}`);
+          }
+        })
+        .then(() => {
+          metricsInterval = setTimeout(fetchMetrics, METRICS_POLL_INTERVAL);
+        })
+        .catch((e) => {
+          console.error(e); // eslint-disable-line no-console
+        });
```
Comment (invincibleJai) [inline frontend/packages/dev-console/src/components/topology/list-view/TopologyListView.tsx:224]: this logic is same as used in Admin side but that's a class based and this is functional connected component https://github.com/openshift/console/blob/master/frontend/public/components/overview/index.tsx#L267 , can we separate it and use across or it'll be beyond scope of this issue WDYT?

Comment (jeff-phillips-18) [reply frontend/packages/dev-console/src/components/topology/list-view/TopologyListView.tsx:224]: I can look into doing this. Eventually we will be replacing the workloads tab contents with the topology list view and the duplication will be removed (see https://github.com/openshift/console/pull/6224)

**Comment by invincibleJai** [comment]
/lgtm

verified the changes, works as expected

**Comment by jeff-phillips-18** [comment]
@invincibleJai squashed and rebased

**Comment by christianvogt** [comment]
Unit test failure related to `observer`

**Pull request merged**